### PR TITLE
Carmen feature 12 page with tips for plant care

### PR DESCRIPTION
**Pull request by carmen136**
**#12 page with tips for plant care**
This feature enables users to read articles about plant care tips which they find on a separate plant tips page.

To achieve this, we:

- Implemented a new page with all plant tips listed that let users have an overview of all available plant care articles
- Implemented a tip details page where a specific article is displayed

### CODE REVIEW

File: components/Navigation/index.js [hunk 1]
```diff
@@ -27,6 +29,11 @@ export default function Navigation() {
                         <IoHeart />
                     </StyledIconContainer>
                 </Link>
+                <Link href="/plantTipsPage" passHref>
```
Comment (F-Kirchhoff) [inline components/Navigation/index.js:32]: you don't need the passHref.

File: pages/tips/[id].js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import TipDetails from "/components/TipDetails/";
+
+export default function TipDetailsPage({ tips }) {
+  const router = useRouter();
+
+  const { id } = router.query;
+
+  const tip = tips.find((tip) => tip.id === id);
+
+  if (!tip) {
```
Comment (F-Kirchhoff) [inline pages/tips/[id].js:11]: when you directly visit this page, the router will not be ready, it won't have the id with the first render. Therefore you will get a flash of your error message. You can prevent this by checking if the router is ready, and if not show a blank page:
```jsx
if(!router.isReady) {
  return null;
}
```

**Review by F-Kirchhoff — APPROVED** [review]
Hey,
this is a great PR so you get an approval. There is one tiny improvement that you can make if you like. You can either way continue with the QA afterwards.

File: pages/tips/[id].js (hunk 2 above)
Comment (carmen136) [reply pages/tips/[id].js:11]: Thanks for the hint! We will implement that

**Pull request merged**